matrix-sdk-android/src/main/java/org/matrix/android/sdk/api/auth/login/LoginWizard.kt

```diff
@@ -54,7 +54,9 @@ interface LoginWizard {
 
     /**
      * Login to the homeserver by sending a custom JsonDict.
-     * The data should contain at least one entry "type" with a String value.
+     * The data should contain:
+     * - at least one entry "type" with a String value.
+     * - a "refresh_token" param to indicate if refresh tokens should be used.
```

Comment:
I think the param (boolean) `refresh_token`, should not be provided here, but added later by the SDK since this is a feature of the SDK to support token refreshing.
So I would not change this doc.

Reply:
makes sense